sdk/keyvault/azure-security-keyvault/tests/test_examples_secrets_async.py

```diff
@@ -0,0 +1,193 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from azure.core.exceptions import ResourceNotFoundError
+from devtools_testutils import ResourceGroupPreparer
+
+from async_preparer import AsyncVaultClientPreparer
+from async_test_case import AsyncKeyVaultTestCase
+
+
+def print(*args):
+    assert all(arg is not None for arg in args)
+
+
+def test_create_secret_client():
+    vault_url = "vault_url"
+    # pylint:disable=unused-variable
+    # [START create_secret_client]
+
+    from azure.identity.aio import AsyncDefaultAzureCredential
+    from azure.security.keyvault.aio.secrets import SecretClient
+
+    # Create a SecretClient using default Azure credentials
+    credentials = AsyncDefaultAzureCredential()
+    secret_client = SecretClient(vault_url, credentials)
```

Comment:
would it be valuable to put a note here, mentioning the required client credentials that need to be present in order for this to work?

Reply:
That would be a long note requiring easy-to-overlook ongoing maintenance. I prefer centralizing explanation in package readmes. I hope the import is enough to point people toward those.

This hope may of course be dashed 😿